packages/trpc/server/routers/viewer/organizations/createTeams.handler.ts

```diff
@@ -49,15 +49,24 @@ export const createTeamsHandler = async ({ ctx, input }: CreateTeamsOptions) =>
 
   const duplicatedSlugs = existingSlugs.filter((slug) => teamNames.includes(slug));
 
-  await prisma.team.createMany({
-    data: teamNames.flatMap((name) => {
+  await prisma.$transaction(
```

Comment:
@leog any benefit of using transaction here?

Reply:
Atomicity mainly, from prisma docs: 
> A database transaction refers to a sequence of read/write operations that are guaranteed to either succeed or fail as a whole.